src/components/Tables/MeetingVotesTable/MeetingVotesTable.tsx

```diff
@@ -1,78 +1,54 @@
 import React from "react";
 import { MeetingVotesTableRow } from "../MeetingVotesTableRow";
 import { EmptyRow } from "../EmptyRow";
+import { ReactiveTable } from "../ReactiveTable";
 import { MeetingVote } from "../../Shared/Types/MeetingVote";
 import "@mozilla-protocol/core/protocol/css/protocol.css";
 
 type MeetingVotesTableProps = {
-  /** the matter being voted upon */
-  votesPage: MeetingVote[];
+  /** an array of Votes */
+  votesPage: object[];
 };
 
 function renderEmpty(index: number) {
-  return <EmptyRow key={`empty-meeting-table-row-${index}`} index={index} />;
+  return <EmptyRow key={`empty-table-row-${index}`} index={index} />;
 }
 
-function renderRows(votesPage: MeetingVote[]) {
-  const isEmpty = !votesPage || !votesPage.length || votesPage.length === 0;
-  return (
-    <tbody>
-      {isEmpty && renderEmpty(0)}
-      {votesPage.map((meetingVotes: MeetingVote, index: number) => {
-        if (!meetingVotes.matter || !meetingVotes.votes) return renderEmpty(index);
-        const legislationName = meetingVotes.matter.name;
-        const legislationDescription = meetingVotes.matter.description;
-        const councilDecision = meetingVotes.council_decision;
-        const legislationLink = `/matters/${meetingVotes.matter.id}`;
-
-        return (
-          <MeetingVotesTableRow
-            key={`meeting-voting-table-row-${index}`}
-            index={index}
-            legislationName={legislationName}
-            legislationDescription={legislationDescription}
-            councilDecision={councilDecision}
-            legislationLink={legislationLink}
-            votes={meetingVotes.votes}
-          />
-        );
-      })}
-    </tbody>
-  );
-}
-
-function renderColGroup() {
-  return (
-    <colgroup>
-      <col style={{ width: "30%" }} />
-      <col style={{ width: "15%" }} />
-      <col style={{ width: "20%" }} />
-      <col style={{ width: "35%" }} />
-    </colgroup>
-  );
-}
-
-function renderHeaders() {
-  return (
-    <thead>
-      <tr>
-        <th scope="col">Legislation</th>
-        <th scope="col">Council Decision</th>
-        <th scope="col">Votes</th>
-      </tr>
-    </thead>
-  );
-}
+const MeetingVotesTable = ({ votesPage }: MeetingVotesTableProps) => {
+  const COLUMN_NAMES = ["Legislation", "Council Decision", "Votes"];
+  const COLUMN_DISTRIBUTION = ["35%", "25%", "40%"];
```

Comment:
The idea here is that Column distribution becomes a prop that will be passed down to lower components.  In the future, we can have user-adjustable column-size that leverages this. I wanted to lay the groundwork here but did not implement it as it would be separate feature.

Reply:
I love the idea of user adjustable column size!